src/ploneintranet/search/results.py

```diff
@@ -1,3 +1,69 @@
 """
 Implementations of the ISearchResult and ISearchResponse
 """
+from zope.interface import implements
+from zope.globalrequest import getRequest
+
+from .interfaces import ISearchResult, ISearchResponse
+
+
+class SearchResult(object):
+    """
+    Data structure containing parsed results from search backend
+    """
+    implements(ISearchResult)
+
+    #: The title of the indexed document (str)
```

Comment:
I think the main thing here is that we don't actually need an interface for the search results at the moment, because they are a consistent data structure that shouldn't need to be reimplemented.